openai-api-examples/src/main/java/bg/codexio/ai/openai/api/examples/thread/CreateThread.java

```diff
@@ -0,0 +1,33 @@
+package bg.codexio.ai.openai.api.examples.thread;
+
+import bg.codexio.ai.openai.api.sdk.thread.Threads;
+
+import java.io.File;
+
+public class CreateThread {
+
+    public static void main(String[] args) {
+        var file = new File(CreateThread.class.getClassLoader()
+                                              .getResource("fake-file.txt")
```

Comment:
Are those fake file uploaded in the resources folder?

Reply:
The file was git ignored , I will add it to the next commit.